packages/lit-virtualizer/web-test-runner.config.js

```diff
@@ -4,11 +4,17 @@
  * SPDX-License-Identifier: BSD-3-Clause
  */
 
+// import {mochaHooks} from './test/benign-errors-root-hooks.js';
```

Comment:
Can this be deleted?

Reply:
Ah yes.  Gone.  Sadly I couldn't get global/root hooks working in browser.